packages/transformer/src/BranchProvenanceInitializer.ts

```diff
@@ -0,0 +1,77 @@
+
+import { ExternalSource, ExternalSourceIsInRepository, IModelDb, RepositoryLink } from "@itwin/core-backend";
+import { DbResult, Id64String } from "@itwin/core-bentley";
+import { Code } from "@itwin/core-common";
+import assert = require("assert");
+import { IModelTransformer } from "./IModelTransformer";
+
+interface ProvenanceInitArgs {
```

Comment:
this should be exported, my bad.

we need to mark every top level export here as @alpha, otherwise I think it's auto public. It's not an official API yet.